letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -333,6 +333,11 @@ elif [ -f /etc/redhat-release ]; then
 
   RPM_DIST_NAME=`(. /etc/os-release 2> /dev/null && echo $ID) || echo "unknown"`
 
+  if [ "$PYVER" -eq 26 -a "$RPM_DIST_NAME" != "rhel" -a $(arch) != 'x86_64' ]; then
```

Comment:
I honestly could not find any way to get a 32 bit version of scl on RHEL and affiliates. I it is available, it is confidential. So I am quite OK to include RHEL in the deprecation.

For arch, well, `uname -m` will do the job!

Reply:
Nice. `uname -m` is indeed in the POSIX standard. Is its output reliable though, at least on RHEL 6 based systems? I tried to find something stating possible outputs but couldn't find anything.

Reply:
I got `x86_64` for my machine for any system I could think of. All code source dumps I could find are stating `x86_64` for 64 bits, and `ix86` (with `x=3,4,5,6`) for 32 bits.

For a standard POSIX tool, I expect this behavior to be extremely consistent.

Reply:
I don't love this. Everything I've seen about `uname` says its output is implementation defined and looking at the source code it seems its output is completely controlled by the kernel where I was unable to find any guarantees about its output across systems/versions.

With that said, I'm not aware of any better options. I considered using Python (which I think we can rely on being there because its used by `yum`), but we can't just check if the machine is 64-bit because [RHEL 6 supports other 64-bit architectures](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/6.0_release_notes/index) and the other options I'm aware of all go back to `uname`.

I think this is _probably_ fine, but I don't like uncertainty like this especially when dealing with certbot-auto.

Reply:
Me neither, in term of solution. But I never saw anything except `x86_64` for 64 bits on kernel architectures.